shipit.yml

```diff
@@ -1,5 +1,7 @@
 deploy:
   override:
     - git submodule update --init --quiet
-    - make install:
-        timeout: 3600
+    - helm dep up ./helm/cas-ggircs
+    - helm upgrade --install --atomic --timeout 900s --namespace "$ENVIRONMENT" --set image.etl.tag "$REVISION" cas-ggircs ./helm/cas-ggircs:
+        timeout: 900
+    - curl -u ${AIRFLOW_USERNAME}:${AIRFLOW_PASSWORD} -X POST https://cas-airflow-${ENVIRONMENT}.pathfinder.gov.bc.ca/api/experimental/dags/ggircs_deploy_db/dag_runs
```

Comment:
any reason to patch this here rather than in `make install`?

Reply:
Nope I can move this to the make install command.

I mainly wanted to check that it was working on shipit. I manually redeployed shipit and added those AIRFLOW_USERNAME and AIRFLOW_PASSWORD env variables to the pod, but I am not completely sure if the shipit jobs can see those env variables (I would assume so)